src/vm/loaderallocator.hpp

```diff
@@ -485,7 +485,6 @@ typedef VPTR(LoaderAllocator) PTR_LoaderAllocator;
 class GlobalLoaderAllocator : public LoaderAllocator
 {
     VPTR_VTABLE_CLASS(GlobalLoaderAllocator, LoaderAllocator)
-    VPTR_UNIQUE_OVERRIDE(VPTRU_LoaderAllocator+1)
```

Comment:
I see, yea having virtual instead of override would be fine I think, at least it would do the expected thing